src/components/InfoModal.tsx

```diff
@@ -89,19 +89,52 @@ function InfoModal(): JSX.Element {
     concentration: concName,
     grad_year: grad,
     auth0_id: id,
+    enroll: enrollYear,
+    planned_grad: plannedGradYear,
   } = info
 
   // Changing information in modal
   const [name, setName] = useState(firstName)
   const nameRef = useRef('')
 
-  const [school, setSchool] = useState(schoolName)
+  const [school, setSchool] = useState(schoolDict[schoolName])
```

Comment:
I believe the issue was fixed in the last push to master. Let's use what's currently in the master branch.